tests/benchmarks/power_consumption/adc/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+zephyr_include_directories(include)
+
+project(app LANGUAGES C)
```

Comment:
use proper project name, without LANGUAGES c

Reply:
project name was changed for all tests